### ISSUE TITLE
splash screen not aligned vertically

### ISSUE THREAD

**Issue by tzapu** [wojtkowiak/meteor-desktop-splash-screen#3]
it appears somewhere in the first third of the screen for me

**Comment by wojtkowiak** [comment]
Is your splash screen image without any unnecessary margins?
It is weird because the windows itself is positioned by passing `center: true` to Chrome and it should always work. The image itself has `'background-position': 'center center',` so it also should be bulletproof. 
Are you using a different configuration than 
```
        "meteor-desktop-splash-screen": {
            "version": "0.0.31",
            "windowSettings": {
                "width": 800,
                "height": 600
            }
        },
```
?

**Comment by tzapu** [comment]
hi, so:
- splash screen png size: 800x600 (so it looks nice on retina displays)
- no margins on image
- splash screen settings 
```
    "meteor-desktop-splash-screen": {
            "version": "0.0.31",
            "windowSettings": {
                "width": 400,
                "height": 300
            }
     },
```
maybe the fact that it's double the resolution moves it up?

**Comment by wojtkowiak** [comment]
I would bet on that. I did not have time yet to test this.
In the meantime if you really want to leave the window settings to this size you can play with:
http://www.w3schools.com/cssref/css3_pr_background-size.asp

You can override the default css in plugin settings: https://github.com/wojtkowiak/meteor-desktop-splash-screen#settings

The default style set is here: https://github.com/wojtkowiak/meteor-desktop-splash-screen/blob/master/src/htmlBody.js#L34

**Comment by tzapu** [comment]
ah, cool, will play with it then mon/tue and get back

thanks

**Comment by wojtkowiak** [comment]
released `0.1.0`

**Comment by tzapu** [comment]
works perfectly for me on macOS now. thanks.
